src/test/java/htsjdk/samtools/SAMRecordTest.java

```diff
@@ -0,0 +1,54 @@
+package htsjdk.samtools;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class SAMRecordTest {
```

Comment:
Unfortunately it looks like `SAMRecordUnitTest` is the class where tests for `SAMRecord` are stored.

Reply:
Thank you, done [here](https://github.com/samtools/htsjdk/pull/994/commits/7843453cf45892d135c824eeac24264bb0db79fa).